### ISSUE TITLE
link rel= detection misses some links

### ISSUE THREAD

**Issue by tmcw** [buttondown/shovel#22]
I think this is the code that's looking for a rel=me twitter account:

https://github.com/buttondown/shovel/blob/a8479357fcc51378b5a2c216c4e2326acb219652/lib/parsers/html.ts#L61

My website does have such a meta tag, but it's all HTML5'd out - uses optional quotes and an implied self-closing tag, so this doesn't get caught:

```html
<link href=https://mastodon.social/@tmcw rel=me>
```

Probably this is one of the more tricky variations to catch, but ideally different attribute order is okay with shovel's parser.

**Comment by jmduke** [comment]
Yup, we should absolutely use `node-html-parser` (which we already have depped) for this. T/Y for flag!
